packages/services/f-stat-client/src/statClient.js

```diff
@@ -0,0 +1,50 @@
+const { Client } = require('@elastic/elasticsearch');
+
+export default class StatClient {
+    constructor (
```

Comment:
Can we make this constructor cleaner having it accept an object instead of so many parameters.  Seven parameter is a code smell to me, it lends itself to confusing code that calls it.